### DISCUSSION TITLE
Agenda for the Flatcar Office Hours on Wednesday, 9th of October 2024, 2:30pm UTC

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1548] [Flatcar Office Hours]
In the office hours we discuss news and happenings in and around Flatcar, provide a space for our user community to discuss with maintainers, and do our release planning.

- Jitsi: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording: https://www.youtube.com/live/NK-5pUaMaZs
  - All recordings: https://www.youtube.com/@FlatcarLinux/live
- Community Calls calendar (all future calls): [Google Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [ical](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)
  
## Welcome
- Brief intro/check-in of the new participants joining the call. Please introduce yourself and share what brings you here today.

## Agenda items
- @ader1990 systemd-256 upgrade and cgroups-v1 deprecation (https://github.com/flatcar/scripts/pull/2145)
- @t-lo will demo ClusterAPI in-place kubernetes updates (https://github.com/flatcar/flatcar-demos/tree/main/CAPO-sysext)

### Docker 26 is now in Beta (4054.1.0), please test!
<details><summary>A number of legacy features have been removed by upstream.</summary>

  [Removed features](https://github.com/docker/cli/blob/master/docs/deprecated.md):
  - [Devicemapper storage driver](https://github.com/docker/cli/blob/master/docs/deprecated.md#device-mapper-storage-driver)
  - [Pushing and pulling with image manifest v2 schema 1](https://github.com/docker/cli/blob/master/docs/deprecated.md#pushing-and-pulling-with-image-manifest-v2-schema-1)
  - [OOM-score adjust for the daemon](https://github.com/docker/cli/blob/master/docs/deprecated.md#oom-score-adjust-for-the-daemon)
  - [logentries logging driver](https://github.com/docker/cli/blob/master/docs/deprecated.md#logentries-logging-driver)
  - [IsAutomated field, and "is-automated" filter on docker search](https://github.com/docker/cli/blob/master/docs/deprecated.md#isautomated-field-and-is-automated-filter-on-docker-search)
  - [Container short ID in network Aliases field](https://github.com/docker/cli/blob/master/docs/deprecated.md#container-short-id-in-network-aliases-field)
  - [legacy API versions](https://github.com/docker/cli/blob/master/docs/deprecated.md#deprecate-legacy-api-versions)
  - [Container and ContainerConfig fields in Image inspect](https://github.com/docker/cli/blob/master/docs/deprecated.md#container-and-containerconfig-fields-in-image-inspect)

</details>

### Release Planning

- https://github.com/orgs/flatcar/projects/7/views/8

### Administrative tasks after meeting concluded
- [x] Create a new Github discussion for the next Flatcar Office Hours: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-office-hours
- [x] Close this discussion as "Resolved".

**Comment by ader1990** [comment]
Hello, I would like to also shortly cover the upgrade of systemd from 255 to 256, upgrade that will have a breaking change - https://github.com/systemd/systemd/releases/tag/v256-rc3:

```
* Support for cgroup v1 ('legacy' and 'hybrid' hierarchies) is now
      considered obsolete and systemd by default will refuse to boot under
      it. To forcibly reenable cgroup v1 support,
      SYSTEMD_CGROUP_ENABLE_LEGACY_FORCE=1 must be set on kernel command
      line. The meson option 'default-hierarchy=' is also deprecated, i.e.
      only cgroup v2 ('unified' hierarchy) can be selected as build-time
      default.
```
